### ISSUE TITLE
 I cant create registration 

### ISSUE THREAD

**Issue by cgsadmin** [wireapp/wire-webapp#8003]
Hello ,
I got below errors when i installed and accessed web 

Access to XMLHttpRequest at 'https://staging-nginz-https.zinfra.io/activate/send' from origin 'http://xx.xx.xx.xx:8081' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

![image](https://user-images.githubusercontent.com/20816988/71893505-e5853800-3171-11ea-8cc8-ec7850bd01ab.png)

_Originally posted by @cgsadmin in https://github.com/wireapp/wire-webapp/pull/7992#issuecomment-571555668_

**Comment by cgsadmin** [comment]
@lipis ,
Please have reply on this so we can proceed to next

**Comment by ffflorian** [comment]
@cgsadmin to run your own instance of the WebApp, you need to set up your own Wire server or start Chrome with `--disable-web-security`. Closing as duplicate of https://github.com/wireapp/wire-webapp/issues/294.
